build_tools/setup_helpers/extension.py

```diff
@@ -132,11 +132,60 @@ def get_ext_modules(debug=False):
             extra_objects=_get_extra_objects(),
             extra_link_args=_get_ela(debug),
         ),
+        _get_transducer_module(),
     ]
 
 
 class BuildExtension(TorchBuildExtension):
     def build_extension(self, ext):
         if ext.name == _EXT_NAME and _BUILD_SOX:
             _build_third_party()
+        if ext.name == _TRANSDUCER_NAME:
+            _build_transducer()
```

Comment:
This is adding `_warp_transducer.so`. This is not the way third-party libraries should be built in torchaudio.
All the third party library should be built as part of `_torchaudio.so` and statically linked into it, so that runtime library management is easy.